appveyor.yml

```diff
@@ -24,7 +24,7 @@ before_build:
     if ($env:GENERATOR) {
       if (!(test-path build)) { mkdir build }
       cd build
-      cmake -G "$env:GENERATOR" .. -DBUILD_SHARED_LIBS=ON -DEVMC_EXAMPLES=ON -DEVMC_TESTING=ON -DCMAKE_INSTALL_PREFIX=C:\install
+      cmake -Wno-dev -G "$env:GENERATOR" .. -DBUILD_SHARED_LIBS=ON -DEVMC_EXAMPLES=ON -DEVMC_TESTING=ON -DCMAKE_INSTALL_PREFIX=C:\install
```

Comment:
Is this `-Wno-dev` supressing the "cmake stack trace" messages?

Reply:
It suppresses warnings like this: https://ci.appveyor.com/project/chfast/evmc/build/243/job/a7d7byb4os8cvm3c
Apparently PowerShell feaks out when it sees any output to standard error.